Flow.Launcher/SettingWindow.xaml

```diff
@@ -2802,7 +2802,13 @@
                                                 Title="{DynamicResource ToggleGameModeHotkey}"
                                                 Icon="&#xe7fc;"
                                                 Type="Inside">
-                                                <cc:HotkeyDisplay Keys="Ctrl+Shift+C" />
+                                                <cc:HotkeyDisplay Keys="Ctrl+F12" />
+                                            </cc:Card>
+                                            <cc:Card
+                                                Title="{DynamicResource RequeryHotkey}"
+                                                Icon="&#xe72c;"
```

Comment:
Can't find a proper icon so this is the same icon as reload plugin data (which is right below).